homework/style.css

```
@@ -1,3 +1,135 @@
+* {
+  margin: 0;
+  padding :0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#root {
+  width: 100%;
+  display: flex;
+  flex-flow: column wrap;
+  background: #f5f5f0;
+  padding: 5px;
+  justify-content: center;
+  
+}
+
+.heading {
+  background: #1a75ff;
+  box-shadow: 2px 5px 7px #000;
+  margin: auto;
+  padding: 30px 2px 2px 12px; 
+  width: 90%;
+  height: 80px;
+  color: #fff;
+}
+
+li {
+  width: 100%;
+  list-style: none;
+  margin: 10px 0 5px 0 ;
+  box-shadow: 2px 5px 7px #000;
```

This suggestion is totally my personal taste, if you disagree then please ignore it 😉 
```suggestion
  box-shadow: 2px 5px 7px #BBB;
```